packages/terra-action-footer/src/_ActionFooterContainer.jsx

```diff
@@ -18,7 +18,7 @@ const ActionFooterContainer = ({
   children,
   ...customProps
 }) => {
-  const isEmpty = !React.Children.count(children);
+  const isEmpty = !React.Children.toArray(children).length;
```

Comment:
Interestingly enough, `React.Children.count` _doesn't_ seem to filter out falsey values. Found this when using the terse syntax of the `ActionFooter`.. when no `start` or `end` was given, it was still rendering with the `.with-actions` class..

The fix: use `React.Children.toArray().length` because
>Returns the children opaque data structure as a flat array
See: [reference](https://reactjs.org/docs/react-api.html#reactchildrentoarray)